python/pythonlab/neighborhood/neighborhood/support/neighborhood_runtime_exception.py

```diff
@@ -0,0 +1,10 @@
+from .neighborhood_signal_key import NeighborhoodSignalKey
+
+
+class NeighborhoodRuntimeException(Exception):
+    def __init__(self, key: NeighborhoodSignalKey):
```

Comment:
Could you explain why you're using `key:  NeighborhoodSignalKey` here? Instead of having a separate `enum` for exceptions as https://github.com/code-dot-org/javabuilder/blob/0ad7f3de06201e90685359ba45ffb0c537d2f1f9/org-code-javabuilder/neighborhood/src/main/java/org/code/neighborhood/support/ExceptionKeys.java#L3, we're planning to combine the keys?

Reply:
updated!